azure-pipelines.yml

```diff
@@ -29,9 +30,12 @@ stages:
       - script: dotnet publish --configuration $(buildConfiguration) --output publish
         displayName: dotnet publish
         workingDirectory: $(directory)
-      # publish artifact for eventual release
-      - publish: $(directory)/publish
-        artifact: app
+      - task: Docker@2
+        displayName: Build Docker Image
+        inputs:
+          repository: 'Dockerfile'
+          command: build
+          Dockerfile: FakebookNotifications
```

Comment:
i said instead of all that. we don't need to build + publish outside docker if the docker build is already a build and publish in its own way. docker build is the only step that needs to be in this whole job (for now, until we get the ACR)

Reply:
also the `inputs` here look wrong